src/commands/git-metadata/__tests__/gitdb.test.ts

```diff
@@ -1153,21 +1428,8 @@ describe('gitdb', () => {
           },
         },
       ],
-      raw: [
-        {
-          input: [
-            'rev-list',
-            '--objects',
-            '--no-object-names',
-            '--filter=blob:none',
-            '--since="1 month ago"',
-            '^87ce64f636853fbebc05edfcefe9cccc28a7968b',
-            '^cc424c261da5e261b76d982d5d361a023556e2aa',
-          ],
-          output: '\n',
-        },
-      ],
-      revparse: [{input: '--is-shallow-repository', output: 'false'}],
+      raw: [],
```

Comment:
ℹ️ we now don't run `rev-list` if there are no commits to include, which I think makes more sense. I might be missing something though

Reply:
before this change: we run `getObjectsToUpload` (the rev-list command) even if `commitsToInclude` was empty. And only then in `generatePackFilesForCommits` we would bail out if `objectsToUpload` was an empty list. 

❓ Can `git rev-list` return something if list of input commits is empty? If the answer is yes, I need to change this

Reply:
this, btw, is also in `dd-trace-js`, so I understand why it'd be here 😄 . If running `git rev-list` with no input commits turns out to be useless (it looks this way testing locally), it'll have to be removed from `dd-trace-js` too.